sdk/openai/openai/samples/cookbook/simpleCompletionsPage/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "azure-openai-browser-completions",
+  "version": "1.0.0",
+  "description": "SPA to demonstrate browser completions using Azure OpenAI",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --mode=production --node-env=production",
+    "build:dev": "webpack --mode=development",
+    "build:prod": "webpack --mode=production --node-env=production",
+    "watch": "webpack --watch"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@azure/identity": "^3.2.2",
```

Comment:
@azure/identity unused? It would be nice to show auth using AAD but I guess configuring it is not trivia.